src/analyticsTracking/amplitudeUserUtils.js

```diff
@@ -0,0 +1,60 @@
+import amplitude from 'amplitude-js';
+import jwtDecode from 'jwt-decode';
+
+/**
+ * convenience class to log user properties on Auth0 login success
+ * @see https://help.amplitude.com/hc/en-us/articles/115002380567-User-Properties-Event-Properties#applying-user-properties-to-events
+ */
+class AmplitudeUser {
+  instance;
+  userId;
+  auth_sub_arr;
+
+  // decoded jwt values
+  auth_user;
+
+  // values to pluck from jwt in the format of [[jwt_prop_name, renmae_to], ... ]
+  user_props = [
+    ['email'],
```

Comment:
We should avoid capturing identifying information like email, names, and and picture. Amplitude will take care of user tracking with its user id.